.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,48 @@
+name: Docker Build
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - 'Dockerfile'
+      - '.github/workflows/docker-build.yml'
+  pull_request:
+    branches: [ main ]
```

Comment:
This means, this Github Action with run if a pull request is made on main. So you can quickly evaluate whether the PR contains obvious breaking changes. 

You can see a history of runs in the “Actions” tab on github. It looks something like this…
<img width="1411" alt="Screenshot 2024-12-26 at 10 15 17 PM" src="https://github.com/user-attachments/assets/1d3a624d-0c3e-4f6c-8657-31610a9bedd4" />